templates/index.html

```diff
@@ -1,93 +1,106 @@
+	<!--loading roboto font -->
 	<link rel='stylesheet' id='font-css'  href='http://fonts.googleapis.com/css?family=Roboto:400,300,100' type='text/css' media='all' />
+
 	<link rel="stylesheet" href="{{url_for('static',filename = 'style.css')}}">
 </head>
 
-
-<div id='wrapper'>
-
-		<div id="tooltip"></div><!-- div to hold tooltip. -->
-		<svg id="statesvg"></svg> <!-- svg to hold the map. -->
-
-		<div id = "cityMap"></div><!-- div to hold map at city level -->
-
-		<!-- the state of music logo - to navigate to home page-->
-		<ul id= "logo">
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/logo.png')}}" alt="the state of music logo" height="100"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/logo2.png')}}" alt="the state of music logo" height="100"></a>
-			</li>
-		</ul>
-
-		<!-- icons to allow toggle between genres - hrefs to be updated -->
-		<ul id = 'navbar'>
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/pop.png')}}" alt="pop music icon" height="105"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/indie.png')}}" alt="indie music icon" height="105"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/country.png')}}" alt="country music icon" height="105"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/classic.png')}}" alt="classic music icon" height="105"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/hiphop.png')}}" alt="hiphop music icon" height="105"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/metal.png')}}" alt="metal music icon" height="105"></a>
-			</li>
-		</ul>
-
-		 <!--about section - text to be updated-->
-		<div id="displayInfo">
-     	<div>
-				<h3>About</h3>
-				<small>
-     			This is some stuff about music in the US.
-					This is some stuff about music in the US.
-					This is some stuff about music in the US.
-					This is some stuff about music in the US.
-					This is some stuff about music in the US.
-					This is some stuff about music in the US.
-					This is some stuff about music in the US.
-     	</div>
-     </div>
-
-		 <div id="displayStats">
-				 <div>
-				 <h3>These states love music:</h3>
-				 <small>
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 </div>
-		 </div>
+<body>
+<div id='container'>
+
+	<!--<div id = "cityMap"></div> div to hold map at city level -->
+
+	<div class="row" id = "navbar">
```

Comment:
I would split the layout into two columns first, one for the sidebar (containing the logo, about, and fun facts) and one for the "main content" (containing the navbar and the map). The sidebar would just contain plain 100% width divs one below the other. And inside the main content column, I would first create a row for the navbar with columns for each music category, and then just a plain div for the map.

Here's a screenshot of the suggested layout:
<img width="1392" alt="Screen Shot 2019-05-01 at 12 02 10 AM" src="https://user-images.githubusercontent.com/929246/56999943-53bc3880-6ba9-11e9-97c0-f8ba872d744e.png">

This would help with two things:
* The sidebar could have its own background colour and be distinct from the "main content" / be scrollable if the fun facts are ridiculously long.
* The map could sit right below the navbar instead of having that awkward space on top of it.